sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -19,6 +19,18 @@
 
 #include <_az_cfg_prefix.h>
 
+
+/**
+ * @brief Azure IoT service MQTT bit field properties for telemetry publish messages.
+ *
+ */
+enum
+{
+  AZ_HUB_CLIENT_DEFAULT_MQTT_TELEMETRY_QOS = 0,

```

Comment:
Telemetry QoS is 1, not 0. (At least according to https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support#using-the-device-sdks)